Source/Core/DolphinQt/Debugger/CodeDiffDialog.cpp

```diff
@@ -133,6 +147,48 @@ void CodeDiffDialog::OnClickItem()
   m_code_widget->SetAddress(address, CodeViewWidget::SetAddressUpdate::WithDetailedUpdate);
 }
 
+void CodeDiffDialog::SaveDataBackup()
+{
+  std::string filename =
+      File::GetUserPath(D_LOGS_IDX) + SConfig::GetInstance().GetGameID() + "_CodeDiff.txt";

```

Comment:
Rather than hardcoding the path this seems like a good use case for DolphinFileDialog. That would also allow for multiple backups per game.

Reply:
Only as long as it doesnt pop up every time you click it. The real world workflow of using this is constant saving, so it would be a huge inconvenience if a file dialog appeared every time. Also, in my opinion multiple for a single game is not very useful.